content/blog/2019/10/2019-10-30-jenkins-performance-avoiding-pitfalls.adoc

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Jenkins Performance: Avoiding Pitfalls, Diagnosing Issues, and Scaling for Growth - DevOps World | Jenkins World 2019"
+tags:
+- jenkins world
+- jenkinsworld
+- devopsworld2019
+- performance
+authors:
+- ryansmith303
+opengraph:
+  image: /images/performance/gc-unhealthy.png
+  image: /images/logos/jenkenstein/256.png
+---
+
+NOTE: This is a speaker blogpost for a DevOps World | Jenkins World 2019 talk in Lisbon, Portugal
+
+With Halloween upon us, there's no better time to discuss Jenkinstein! Are you suffering from Jenkins Performance issues? Are users complaining about a slow UI or even scarier, Jenkins is frequently going down?  
+
+image::/images/logos/jenkinstein/256.png[]
+
+Come join me at link:https://www.cloudbees.com/devops-world/lisbon[DevOps World | Jenkins World 2019] for "link:https://sched.co/UVVp[**Jenkins Performance: Avoiding Pitfalls, Diagnosing Issues, and Scaling for Growth**]", a talk about JVM administration, and best practices from the front lines of supporting thousands of Jenkins installations worldwide. During this talk we'll cover how to prevent your Jenkins instance from becoming a Jenkinstein! 
+
+Topics we will be discussing:
+
+* JVM Administration Best Practices
+* Horizontal Scaling
+* Analyzing Thread Dumps, GC Logs, and Heap Dumps
+* Real World Data showing 3500% Performance Increases
+* Garbage Collection
+
+image::/images/performance/gc-unhealthy.png[]
+
+We'll be covering how by analyzing spooky data like Garbage Collection logs, you can forecast for growth, and baseline using key performance indicators like application throughput and latency! 
```

Comment:
I would reverse this sentence's structure:

"We'll be covering how you can forecast for growth, and baseline using key performance indicators like application throughput and latency, by analyzing spooky data like Garbage Collection logs!"

And I might say "discussing" instead of "covering" but whatever.

Reply:
Changes applied.